### PR TITLE
run rustfmt on liballoc_jemalloc folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/liballoc_jemalloc/build.rs [hunk 1]
```diff
@@ -33,18 +33,26 @@ fn main() {
                  jemalloc.parent().unwrap().display());
         let stem = jemalloc.file_stem().unwrap().to_str().unwrap();
         let name = jemalloc.file_name().unwrap().to_str().unwrap();
-        let kind = if name.ends_with(".a") {"static"} else {"dylib"};
+        let kind = if name.ends_with(".a") {
+            "static"
+        } else {
+            "dylib"
+        };
```
Comment (nagisa) [inline src/liballoc_jemalloc/build.rs]: The previous version was much better here.

Comment (srinivasreddy) [inline src/liballoc_jemalloc/build.rs]: cc @nrc . In the If expression. If we return static (?) strings  and the total if expression fits  under 100 characters, then we format the code like this.
`let kind = if name.ends_with(".a") { "static" } else { "dylib" };`
what do you say?

**Comment by brson** [comment]
@nagisa Can we go ahead and land this or do you want that formatting fixed here?

**Comment by nagisa** [comment]
Its fine I guess. Eventually somebody else will come along and submit a PR with the same thing in it anyway.

@bors r+

**Pull request merged**